plugins/hydra_nevergrad_sweeper/tests/test_nevergrad_sweeper_plugin.py

```diff
@@ -1,8 +1,10 @@
 # Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+import subprocess
 from pathlib import Path
 from typing import Any
 
 import nevergrad as ng
+import pytest
```

Comment:
could we do `from pytest import mark` instead to be consistent with the rest of the repo?

Reply:
So `from pytest import mark, raises`?
I've reverted to importing `pytest` since I am using those two functions, and I felt it would be more structured/simpler to follow this way, but I'll do what you prefer.

Reply:
Yes, pls import them. This is the [commit](https://github.com/facebookresearch/hydra/commit/9bf7800157692795090f3695efe136bbbd6fef1d) that modifies the whole repo to import the functions.

Reply:
done, and i'm not using raises anymore anyway.
out of curiosity, what was the reason for this move? on my side I do prefer keeping the namespace because it feels clearer to me, but that's a personal choice I guess.